build-system/tasks/cherry-pick.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const log = require('fancy-log');
+const {getOutput} = require('../common/exec');
+const {green, cyan, red, yellow} = require('ansi-colors');
+
+/**
+ * Executes a shell command, and logs an error message if the command fails.
+ *
+ * @param {string} cmd
+ * @param {string} msg
+ * @return {!Object}
+ */
+function execOrLog(cmd, msg) {
```

Comment:
The name is a bit confusing - it should probably be `execOrThrow`. Also, this should be moved to exec.js to avoid someone else not realizing it's already implemented and re-implementing it in their own task :)

Reply:
https://github.com/ampproject/amphtml/pull/28614#discussion_r432146525